profiles/management/tests.py

```diff
@@ -0,0 +1,82 @@
+"""
+Tests for management commands for profiles
+"""
+from django.contrib.auth.models import User
+
+from courses.factories import CourseRunFactory
+from courses.models import (
+    Course,
+    CourseRun,
+    Program,
+)
+from dashboard.models import (
+    CachedCertificate,
+    CachedEnrollment,
+    ProgramEnrollment,
+)
+from profiles.factories import UserFactory
+from profiles.management.commands import (
+    gen_realistic_search_data,
+    delete_realistic_search_data,
+)
+from roles.models import (
+    Role,
+    Staff,
+)
+from search.base import MockedESTestCase
+
+
+# pylint: disable=no-self-use
+class FakeDataTests(MockedESTestCase):
+    """
+    Tests regarding fake data
+    """
+
+    def test_can_generate_fake_data(self):
+        """
+        Test that the management command generates fake data
+        """
+        gen_realistic_search_data.Command().handle()
+
+        assert Program.objects.count() == 2
+        assert Course.objects.count() == 6
+        assert CourseRun.objects.count() == 18
+        assert Program.objects.filter(
+            description__startswith=delete_realistic_search_data.FAKE_PROGRAM_DESC_PREFIX
+        ).count() == 2
+        assert Course.objects.count() == 6
+        assert CourseRun.objects.count() == 18
+        # No staff option provided
+        assert Role.objects.count() == 0
+        assert ProgramEnrollment.objects.count() == 60
+        assert User.objects.count() == 60
+        assert User.objects.filter(
+            username__startswith=delete_realistic_search_data.FAKE_USER_USERNAME_PREFIX
+        ).count() == 60
+        assert CachedEnrollment.objects.count() == 60
+        assert CachedCertificate.objects.count() == 60
```

Comment:
these two assertions pass for you?? they certainly don't for me, and i don't think they should. our current approach is to create a CachedEnrollment and CachedCertificate for every single CourseRun times every single User (most of which just have a null `data` value). this count is 1080 when i run this test suite (60 users * 18 course runs). even if you're just looking for non-'empty' Certificates/Enrollments, i'm pretty sure this logic is wrong. some users have several Certificates/Enrollments, some have none at all, and i doubt that they both add up to 60 total